hammer/par/innovus/__init__.py

```diff
@@ -483,6 +486,9 @@ def place_pins(self) -> bool:
                     assign_arg = "-assign {{ {x} {y} }}".format(x=pin.location[0], y=pin.location[1])
 
                 layers_arg = ""
+                if set(pin.layers).intersection(set(power_pin_layers)):
```

Comment:
One of these things in the set is Optional - may need a `get_or_else`

Reply:
Fixed.